### PR TITLE
Update HomeController.cs

### PR DESCRIPTION
**Pull request by meaghanlewis**
Add another message in the home controller.

**v important**

# BIG BIG BIG

> test PR

[https://github.com/mastahyeti/OctoGAS](url)

- 1
- 2
- 3

1. one
2. two
3. three

### CODE REVIEW

File: TestWebAppProject/Controllers/HomeController.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ public IActionResult Index()
         public IActionResult About()
         {
             ViewData["Message"] = "Your application description page.";
+            ViewData["Message"] = "Your application body.";
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs]: 1

Comment (meaghanlewis) [reply TestWebAppProject/Controllers/HomeController.cs]: replying

Comment (meaghanlewis) [reply TestWebAppProject/Controllers/HomeController.cs]: <html>
<head>
<title>Title of the document</title>
</head>

<body>
The content of the document......
</body>

</html>

File: TestWebAppProject/Controllers/HomeController.cs [hunk 2]
```diff
@@ -16,6 +16,7 @@ public IActionResult Index()
...
 
             return View();
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs]: https://github.com/github/VisualStudio/issues/1159

File: TestWebAppProject/Controllers/HomeController.cs [hunk 3]
```diff
@@ -16,6 +16,7 @@ public IActionResult Index()
         public IActionResult About()
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs]: comment

File: TestWebAppProject/Controllers/HomeController.cs [hunk 4]
```diff
@@ -16,6 +16,7 @@ public IActionResult Index()
...
         {
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs]: 1

File: TestWebAppProject/Controllers/HomeController.cs [hunk 5]
```diff
@@ -16,6 +16,7 @@ public IActionResult Index()
...
             ViewData["Message"] = "Your application description page.";
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs]: 2

File: TestWebAppProject/TestWebAppProject.csproj [hunk 6]
```diff
@@ -7,11 +7,11 @@
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: 1

Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: 1

Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: comment

File: TestWebAppProject/appsettings.Development.json [hunk 7]
```diff
@@ -2,7 +2,7 @@
   "Logging": {
```
Comment (meaghanlewis) [inline TestWebAppProject/appsettings.Development.json:2]: comment here

File: TestWebAppProject/TestWebAppProject.csproj (hunk 6 above)
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: comment here

Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: comment on line 7

File: TestWebAppProject/TestWebAppProject.csproj [hunk 8]
```diff
@@ -7,11 +7,11 @@
 
   <ItemGroup>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: comment on line 8

File: TestWebAppProject/Controllers/HomeController.cs [hunk 9]
```diff
@@ -16,6 +16,7 @@ public IActionResult Index()
...
+            ViewData["Message"] = "Your application body.";
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs]: comment on line 20

File: TestWebAppProject/appsettings.Development.json [hunk 10]
```diff
@@ -2,7 +2,7 @@
...
     "IncludeScopes": false,
     "LogLevel": {
-      "Default": "Debug",
+      "Default": "Error",
       "System": "Information",
       "Microsoft": "Information"
     }
```
Comment (meaghanlewis) [inline TestWebAppProject/appsettings.Development.json:8]: blaha

File: TestWebAppProject/TestWebAppProject.csproj [hunk 11]
```diff
@@ -7,11 +7,11 @@
...
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: line 10

File: TestWebAppProject/TestWebAppProject.csproj [hunk 12]
```diff
@@ -7,11 +7,11 @@
...
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.2" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.3" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.1.2" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.2" />
   </ItemGroup>
```
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj]: line 15

File: TestWebAppProject/TestWebAppProject.csproj (hunk 6 above)
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj:7]: comment on line 7

File: TestWebAppProject/TestWebAppProject.csproj (hunk 11 above)
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj:10]: comment on line 10. can edit?

File: TestWebAppProject/Controllers/HomeController.cs [hunk 13]
```diff
@@ -16,6 +16,7 @@ public IActionResult Index()
...
 
             return View();
         }
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs]: comment on line 22

File: TestWebAppProject/TestWebAppProject.csproj [hunk 14]
```diff
@@ -7,11 +7,11 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" />
```
Comment (ghfvs-tester) [inline TestWebAppProject/TestWebAppProject.csproj:9]: start a review

File: TestWebAppProject/Views/Shared/Error.cshtml [hunk 15]
```diff
@@ -12,3 +12,11 @@
 <p>
     <strong>Development environment should not be enabled in deployed applications</strong>, as it can result in sensitive information from exceptions being displayed to end users. For local debugging, development environment can be enabled by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong>, and restarting the application.
 </p>
+
```
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Shared/Error.cshtml:15]: looks good!

File: TestWebAppProject/appsettings.Development.json [hunk 16]
```diff
@@ -2,7 +2,7 @@
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
-      "Default": "Debug",
+      "Default": "Error",
```
Comment (ghfvs-tester) [inline TestWebAppProject/appsettings.Development.json:5]: error?

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]
https://ci.appveyor.com/project/github-windows/visualstudio/build/2.4.99.2033/artifacts

File: TestWebAppProject/Views/Shared/Error.cshtml [hunk 17]
```diff
@@ -12,3 +12,11 @@
...
+<h3>Development Mode</h3>
```
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Shared/Error.cshtml:16]: error1

File: TestWebAppProject/Views/Shared/Error.cshtml [hunk 18]
```diff
@@ -12,3 +12,11 @@
...
+<p>
```
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Shared/Error.cshtml:17]: error2

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — COMMENTED** [review]
lgtm

File: TestWebAppProject/appsettings.Development.json [hunk 19]
```diff
@@ -2,7 +2,7 @@
...
       "System": "Information",
       "Microsoft": "Information"
```
Comment (ghfvs-tester) [inline TestWebAppProject/appsettings.Development.json:7]: how about "microsoft" -> ~cross~

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]

File: TestWebAppProject/Views/Shared/Error.cshtml [hunk 20]
```diff
@@ -12,3 +12,11 @@
...
+    Swapping to <strong>Development</strong> environment will display more detailed information about the error that occurred.
```
Comment (ghfvs-tester) [inline TestWebAppProject/Views/Shared/Error.cshtml:18]: <strong>dev</strong>

**Review by meaghanlewis — COMMENTED** [review]
can i comment here

File: TestWebAppProject/Views/Shared/Error.cshtml (hunk 15 above)
Comment (meaghanlewis) [reply TestWebAppProject/Views/Shared/Error.cshtml]: reply

File: TestWebAppProject/TestWebAppProject.csproj (hunk 14 above)
Comment (meaghanlewis) [inline TestWebAppProject/TestWebAppProject.csproj:9]: start a new review

File: TestWebAppProject/Controllers/HomeController.cs [hunk 21]
```diff
@@ -11,6 +11,7 @@ public class HomeController : Controller
         public IActionResult Index()
         {
             return View();
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:13]: add new comment

File: TestWebAppProject/Controllers/HomeController.cs [hunk 22]
```diff
@@ -11,6 +11,7 @@ public class HomeController : Controller
...
+            return View();
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:14]: new pending comment!!!!

File: TestWebAppProject/Controllers/HomeController.cs (hunk 21 above)
Comment (ghfvs-tester) [inline TestWebAppProject/Controllers/HomeController.cs:13]: start a new review

File: TestWebAppProject/Controllers/HomeController.cs [hunk 23]
```diff
@@ -11,6 +11,7 @@ public class HomeController : Controller
...
         }
```
Comment (ghfvs-tester) [inline TestWebAppProject/Controllers/HomeController.cs:15]: comment 2

File: TestWebAppProject/Controllers/HomeController.cs [hunk 24]
```diff
@@ -11,6 +11,7 @@ public class HomeController : Controller
...
```
Comment (ghfvs-tester) [inline TestWebAppProject/Controllers/HomeController.cs:16]: add another

Comment (ghfvs-tester) [reply TestWebAppProject/Controllers/HomeController.cs:16]: reply

**Review by ghfvs-tester — APPROVED** [review]

File: TestWebAppProject/TestWebAppProject.csproj [hunk 25]
```diff
@@ -7,11 +7,11 @@
...
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.2" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.3" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.1.2" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.2" />
   </ItemGroup>
 
   <ItemGroup>
```
Comment (ghfvs-tester) [inline TestWebAppProject/TestWebAppProject.csproj:17]: new review

Comment (ghfvs-tester) [reply TestWebAppProject/TestWebAppProject.csproj:17]: reply

Comment (ghfvs-tester) [reply TestWebAppProject/TestWebAppProject.csproj:17]: reply2

File: TestWebAppProject/TestWebAppProject.csproj [hunk 26]
```diff
@@ -7,11 +7,11 @@
...
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.2" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.1.2" />
```
Comment (ghfvs-tester) [inline TestWebAppProject/TestWebAppProject.csproj:12]: How about this?

**Review by ghfvs-tester — APPROVED** [review]

File: TestWebAppProject/appsettings.Development.json [hunk 27]
```diff
@@ -2,7 +2,7 @@
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
-      "Default": "Debug",
+      "Default": "Error",
       "System": "Information",
```
Comment (meaghanlewis) [inline TestWebAppProject/appsettings.Development.json:6]: new review

**Review by meaghanlewis — COMMENTED** [review]
lgtm

File: TestWebAppProject/appsettings.Development.json (hunk 27 above)
Comment (meaghanlewis) [reply TestWebAppProject/appsettings.Development.json:6]: single comment

File: TestWebAppProject/Controllers/HomeController.cs (hunk 21 above)
Comment (meaghanlewis) [reply TestWebAppProject/Controllers/HomeController.cs]: reply

Comment (meaghanlewis) [reply TestWebAppProject/Controllers/HomeController.cs]: reply

**Pull request merged**